### PR TITLE
Checkbox fix

### PR DESCRIPTION
**Pull request by Lillian-wang**
Fix the bug: Theme 2 - check boxes broken #354

### CODE REVIEW

File: css/templates/themes/2/page-vars.css [hunk 1]
```diff
@@ -50,7 +50,7 @@
   --color-navbar-primary: var(--color-black);
   --color-navbar-font-primary: var(--color-white);
 
-  --color-input-primary: var(--color-font-accent);
+  --color-input-primary: var(--color-secondary);
```
Comment (camelburrito) [inline css/templates/themes/2/page-vars.css]: Changing the primary color will break other inputs , can you try to set the right secondary color (i have mentioned the variables in the steps)

Comment (Lillian-wang) [inline css/templates/themes/2/page-vars.css]: So I should keep " --color-input-primary: var(--color-font-accent);"  and add a new line: " --color-input-secondary: var(--color-secondary); " , right?

Comment (camelburrito) [inline css/templates/themes/2/page-vars.css]: yes , it should be a color that fits the theme (i don't know what it is until i download your patch and check - if you can upload an img i can verify)

Comment (Lillian-wang) [inline css/templates/themes/2/page-vars.css]: Ok updated. PTAL

**Comment by camelburrito** [comment]
@Lillian-wang - do you have screenshots of how it looks after your fix?

**Comment by Lillian-wang** [comment]
@camelburrito , can you specify what screenshot you would like to have? Since I am just adding some attributes onto links, the site looks exactly the same as before.

**Comment by camelburrito** [comment]
@Lillian-wang - this is the checkbox bug you fixed not the links one :)

**Comment by Lillian-wang** [comment]
@camelburrito , for some reason, it worked when I fixed the bug last week. But now it doesn't work again. I will take a look later.

**Comment by Lillian-wang** [comment]
@camelburrito 
screen shot 2017-05-29 at 11 53 21 pm

**Comment by camelburrito** [comment]
@Lillian-wang This looks great!  Thanks!

**Pull request merged**